luthier/config/src/utils/defaultingUtils.lua

```diff
@@ -15,6 +15,7 @@ M.DEFAULT_DIMENSION_CATEGORY = "UNKNOWN_CATEGORY"
 M.DEFAULT_DIMENSION_IS_AGGREGATABLE = true
 M.DEFAULT_DIMENSION_FIELDS = {}
 M.DEFAULT_DIMENSION_KEY_VALUE_STORE = "memory"
+M.DEFAULT_DIMENSION_SKIP_LOADING = true
```

Comment:
I don't think we should make the system skip dimension loading by default. If people are going to
be skipping a key part of the dimension infrastructure, they should do it explicitly. Besides, if
somebody isn't using the dimension loading at all, there's nothing stopping them from defining
their own defaults.

Reply:
done.